person.rb

```diff
@@ -0,0 +1,27 @@
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
+class Person < Nameable
+  attr_accessor :age, :name, :id
+
+  def initialize(age, id, name = 'unknown', parent_permission: true)
+    super()
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
Kindly make sure that you have `@rentals`  variable (in your case `@rental_details`) initiated in the `Person` class as well.